settings.gradle.kts

```diff
@@ -1,3 +1,7 @@
+plugins {
+    id("org.gradle.toolchains.foojay-resolver-convention") version "0.7.0"
```

Comment:
Is that required? Seems like it's not using

Reply:
It's not required, just a convenience for the contributors.
I used it for Windows to avoid manually installing required JDKs. Thought, that it might be useful in general, to let gradle automatically resolve and download JDKs. What do you think?

Reply:
I finally got it.
Gradle documentation looks confusing and I thought that this plugin requires some configuration, but as far as I understand "convention" plugins means having some default settings.
Sorry for misunderstanding.